modules/dashboards/application/controllers/IndexController.php

```diff
@@ -14,35 +15,38 @@ class IndexController extends Controller
 
     public function indexAction()
     {
-        $this->createTabs();
-
         $select = (new Select())
             ->columns('dashlet.name, dashlet.dashboard_id, dashlet.url')
             ->from('dashlet')
             ->join('dashboard d', 'dashlet.dashboard_id = d.id')
-            ->where(['d.id = ?' => $this->getTabs()->getActiveName()]);
+            ->where(['d.id = ?' => $this->createTabsAndAutoActivateDashboard()]);
 
         $dashlets = $this->getDb()->select($select);
 
         $this->content = new DashboardWidget($dashlets);
     }
 
-    protected function createTabs()
+    /**
+     * create Tabs and
+     * activate the first dashboard from the database when the url doesn't have a parameter
+     * @return int $id
```

Comment:
It seems Eric told you already often enough what to look out for when writing inline documentation? You'll then certainly know what's wrong here. :wink:

Reply:
Well, minus the superfluous `$id`. You've fixed that already :sweat_smile: